pkg/router/functionHandler.go

```diff
@@ -494,6 +508,17 @@ func (fh functionHandler) handler(responseWriter http.ResponseWriter, request *h
 		rrt.closeContext()
 	}()
 
+	// add attributes to current span
+	ctx := request.Context()
+	span := trace.SpanFromContext(ctx)
+	attributes := []attribute.KeyValue{
```

Comment:
can we move this otelutils ?
```sh
func GetAttributesForFunction(fn *fv1.Function) []attribute.KeyValue
```

I assume we would need to add these attributes in executor and fetcher also ? Or these are retained in next spans ?